nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/CleanSurefireExecution.java

```diff
@@ -120,12 +120,21 @@ protected void setupArgline() {
                 this.originalArgLine + " " + this.configuration.toArgLine());
     }
 
-    private Xpp3Dom setReportOutputDirectory(Xpp3Dom configuration) {
+    private Xpp3Dom applyNonDexConfig(Xpp3Dom configuration) {
         Xpp3Dom configNode = configuration;
         if (configNode == null) {
             configNode = new Xpp3Dom("configuration");
         }
 
+        configNode = addExcludedGroups(configNode);
+        return setReportOutputDirectory(configNode);
+    }
+
+    private Xpp3Dom addExcludedGroups(Xpp3Dom configNode) {
+        return this.addAttributeToConfig(configNode, "excludedGroups", "edu.illinois.NonDexIgnore");
```

Comment:
What happens if the pom already had excludeGroups, e.g., excludeGroups=foo.bar.LongRunning? Maybe you should concatenate rather than replace the value? Please add an integration test illustrating this case to make sure it works.